mapsTask/js/service.js

```diff
@@ -0,0 +1,39 @@
+var companyStorage = (function () {
+    function Company(name, address, email, phone, web) {
+        this.name = name;
+        this.address = address;
+        this.email = email;
+        this.phone = phone;
+        this.web = web;
+    };
+
+    function CompanyStorage() {
+        if (localStorage.getItem("allCompanies") != null) {
```

Comment:
For comparison, using triple equals is recommended. Use `!==` instead of `!=` please. Find out the difference and why it's more appropriate.